src/vsr.zig

```diff
@@ -1097,6 +1097,47 @@ test "quorums" {
     }
 }
 
+/// Deterministically assigns replica_ids for the initial configuration.
+///
+/// Eventualy, we want to identify replicas using random u128 ids to prevent operator errors.
+/// However, that requires unergonomic two-step process for spinning a new cluster up.  To avoid
+/// needlessly compromizing the experience until reconfiguration is fully implemented, derive
+/// replica ids for the initial cluster deterministically.
+pub fn initial_configuration(cluster: u32) [constants.nodes_max]u128 {
```

Comment:
Debating whether I should introduce 

```zig
const Configuration = struct {
  replicas: [constants.nodes_max]u128,
  replica_count: u8,
}
```

already.

The main reason why I don't want to do that is that in the VSR header we want to manually align the fields.